packages/contracts/contracts/core/SphinxModule.sol

```diff
@@ -34,7 +34,7 @@ contract SphinxModule is ReentrancyGuard, Enum, ISphinxModule, Initializable {
     /**
      * @inheritdoc ISphinxModule
      */
-    string public constant override VERSION = "1.0.0";
+    string public constant override VERSION = "1.0.0"; // TODO(later): what version?
```

Comment:
IMO, this should be a major version change since the data structure of the Merkle tree is changing.